HISTORY.md

```diff
@@ -1,5 +1,7 @@
 # Rocksdb Change Log
 ## Unreleased
+### Bug fixes
+* Reset the `refitting_level_` flag to false in the error paths in `ReFitLevel()`. Without this fix, if ReFitLevel() returns error once, all subsequent calls to it will result in 'Not implemented: another thread is refitting' error.
```

Comment:
Proposed alternative in terms of the relevant user-level APIs.
```
* Fixed a bug after a `CompactRange()` with `CompactRangeOptions::change_level` set fails due to a conflict in the level change step, which caused all subsequent calls to `CompactRange()` with `CompactRangeOptions::change_level` set to incorrectly fail with a `Status::NotSupported("another thread is refitting")` error.
```

Reply:
Thanks Andrew. Fixed...